.circleci/config.yml

```diff
@@ -7,8 +17,182 @@ orbs:
 
 
 jobs: 
+   build-and-push-docker-image:
+    environment:
+      - IMAGE: taraxa-node
+#      - CIRCLE_PULL_REQUEST: << pipeline.parameters.pull_request >>
+    machine:
+      image: ubuntu-1604:202007-01
```

Comment:
I would update the image to: `ubuntu-2004:202104-01`
since 1604 is not updated anymore